multi-agent/lib/mas/elements/nodes/claude_agent/card_builder.py

```
@@ -0,0 +1,63 @@
+"""
+Card builder for Claude Agent Node.
+
+Builds element card from local configuration (model, tools, capabilities).
+"""
+
+from typing import List
```

Ruff (0.15.15)

[warning] `typing.List` is deprecated, use `list` instead

(UP035)